runtime/src/accounts_index.rs

```diff
@@ -136,7 +163,7 @@ mod tests {
         let key = Keypair::new();
         let index = AccountsIndex::<bool>::default();
         let ancestors = HashMap::new();
-        assert_eq!(index.get(&key.pubkey(), &ancestors), None);
+        assert!(index.get(&key.pubkey(), &ancestors).is_none());
```

Comment:
I'm curious why you changed all these `assert_eq`.  The nice thing about `assert_eq` here instead of `assert` is that when it fails, `assert_eq` prints out the LHS which makes triage much easier

Reply:
RightGuard thingy is not printable

Reply:
it is if you deref it?

Reply:
I'll try that.

Reply:
```
error[E0369]: binary operation `==` cannot be applied to type `std::option::Option<(std::sync::RwLockReadGuard<'_, std::v
ec::Vec<(u64, bool)>>, usize)>`
   --> runtime/src/accounts_index.rs:171:9
    |
171 |         assert_eq!(index.get(&key.pubkey(), &ancestors), None);
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |         |
    |         std::option::Option<(std::sync::RwLockReadGuard<'_, std::vec::Vec<(u64, bool)>>, usize)>
    |         std::option::Option<_>
    |
    = note: an implementation of `std::cmp::PartialEq` might be missing for `std::option::Option<(std::sync::RwLockReadGu
ard<'_, std::vec::Vec<(u64, bool)>>, usize)>`
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-
backtrace for more info)
```

Reply:
```
error[E0614]: type `std::option::Option<(std::sync::RwLockReadGuard<'_, std::vec::Vec<(u64, bool)>>, usize)>` cannot be dereferencedi
ld failed, waiting for other jobs to finish...
   --> runtime/src/accounts_index.rs:171:20
    |
171 |         assert_eq!(*index.get(&key.pubkey(), &ancestors), None);
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```